crypto/crypto_test.go

```diff
@@ -0,0 +1,48 @@
+package crypto
+
+import (
+	"testing"
+)
+
+func Test_NewEcSha256Signer_WithIncorrectKey_ShouldGetError(t *testing.T) {
+	if _, err := NewEcSha256Signer([]byte("not a key")); err == nil {
+		t.Errorf("should get an error")
+	}
+}
+
+func Test_NewEcSha256Signer_WithCorrectKey_ShouldGetInstance(t *testing.T) {
+	var key string = "-----BEGIN EC PRIVATE KEY-----\n" +
+		"MHcCAQEEICcJGMEw3dyXUGFu/5a36HqY0ynZi9gLUfKgYWMYgr/IoAoGCCqGSM49\n" +
+		"AwEHoUQDQgAEBGuhqumyh7BVNqcNKAQQipDGooUpURve2dO66pQCgjtSfu7lJV20\n" +
+		"XYWdrgo0Y3eXEhvK0lsURO9N0nrPiQWT4A==\n-----END EC PRIVATE KEY-----\n"
+
+	if _, err := NewEcSha256Signer([]byte(key)); err != nil {
+		t.Errorf("should get an correct signer instance")
+	}
+}
+
+func Test_EcSha256Signer_WithCorrectKey_ShouldSignCorrectly(t *testing.T) {
+	var key string = "-----BEGIN EC PRIVATE KEY-----\n" +
+		"MHcCAQEEICcJGMEw3dyXUGFu/5a36HqY0ynZi9gLUfKgYWMYgr/IoAoGCCqGSM49\n" +
+		"AwEHoUQDQgAEBGuhqumyh7BVNqcNKAQQipDGooUpURve2dO66pQCgjtSfu7lJV20\n" +
+		"XYWdrgo0Y3eXEhvK0lsURO9N0nrPiQWT4A==\n-----END EC PRIVATE KEY-----\n"
+	var s EcSha256Signer
+	var err error
+	var signed []byte
+
+	if s, err = NewEcSha256Signer([]byte(key)); err != nil {
+		t.Errorf("should get an correct signer instance")
+	}
+
+	if signed, err = s.Sign([]byte("hello world!")); err != nil {
+		t.Errorf("should be able to sign")
+	}
```

Comment:
I think we can use `func VerifyASN1` from `ecdsa` to verify the signed bytes here

Reply:
I will add this to the end of this test case.

Reply:
@thongdk8 

Thank you for your advice.
I added other assertions https://github.com/scalar-labs/scalardl-go-client-sdk/pull/2/commits/18a361ddfcaeed6d5f560b97a29a33c99d1bb836

Please take a look if you are interested :D